analytics/lib/counts.py

```diff
@@ -851,6 +851,8 @@ def get_count_stats(realm: Realm | None = None) -> dict[str, CountStat]:
             ),
             CountStat.DAY,
         ),
+        # AI credit usage stats for users.
+        LoggingCountStat("ai_credit_usage::hour", UserCount, CountStat.HOUR),
```

Comment:
Do we want this to be daily rather than hourly? It's a lot less storage used, and I don't think we have a reason to particularly need hourly metrics/graphs of AI usage; we just care about this for enforcement, and daily suffices for that.

Reply:
Okay. Changed.